unison-runtime/src/Unison/Runtime/Foreign/Function.hs

```diff
@@ -200,6 +205,11 @@ import Unison.Util.Text qualified as Util.Text
 import Unison.Util.Text.Pattern qualified as TPat
 import UnliftIO qualified
 
+withMutableByteArrayContents :: (PA.PrimBase m) => PA.MutableByteArray (PA.PrimState m) -> (Ptr Word8 -> m a) -> m a
+{-# INLINE withMutableByteArrayContents #-}
+withMutableByteArrayContents mba =
+  PA.keepAlive (PA.mutableByteArrayContents mba)
```

Comment:
So, I don't understand what's going on here. It doesn't look like it should even type check. Is Github showing me the wrong diff or something?